code/datums/helper_datums/getrev.dm

```diff
@@ -21,13 +25,70 @@ var/global/datum/getrev/revdata = new()
 	if(testmerge.len)
 		world.log << commit
 		for(var/line in testmerge)
-			if(line)
-				world.log << "Test merge active of PR #[line]"
+			world.log << "Test merge active of PR #[line]"
 		world.log << "Based off master commit [parentcommit]"
 	else
 		world.log << parentcommit
 	world.log << "Current map - [MAP_NAME]" //can't think of anywhere better to put it
 
+/datum/getrev/proc/DownloadPRDetails()
+	if(!config.githubrepoid)
+		if(testmerge.len)
+			world.log << "PR details download failed: No github repo config set"
+		return
+	if(!isnum(text2num(config.githubrepoid)))
+		world.log << "PR details download failed: Invalid github repo id: [config.githubrepoid]"
+		return
+	for(var/line in testmerge)
+		//"This has got to be the ugliest hack I have ever done"
+		//warning, here be dragons
+		/*
+			                |  @___oo
+			      /\  /\   / (__,,,,|
+			     ) /^\) ^\/ _)
+			     )   /^\/   _)
+			     )   _ /  / _)
+			 /\  )/\/ ||  | )_)
+			<  >      |(,,) )__)
+			 ||      /    \)___)\
+			 | \____(      )___) )___
+			  \______(_______;;; __;;;
+			*/
+		if(!isnum(text2num(line)))
+			world.log << "PR details download failed: Invalid PR number: [line]"
+			return
+		var/url = "https://api.github.com/repositories/[config.githubrepoid]/pulls/[line].json"
+		var/temp_file = "prcheck[line].json"
+		var/command = "powershell -Command \"curl -O [temp_file] [url]\""
+		var/result = shell(command)
+		if(result != 0)
+			world.log << "PR details download failed: shell exited with code: [result]"
+			return
+
+		var/f = file(temp_file)
+		if(!f)
+			world.log << "PR details download failed: Temp file not found"
+			return
+		testmerge[line] = json_decode(file2text(f))
+		f = null
+		fdel(temp_file)
+
+		if(!testmerge[line])
+			world.log << "PR details download failed: null details returned"
+			return
+		CHECK_TICK
+	has_pr_details = TRUE
+
+/datum/getrev/proc/GetTestMergeInfo()
+	if(!testmerge.len)
+		return ""
+	. = "The following pull requests are currently test merged:<br><br/>"

```

Comment:
stop with the `/>`

Byond doesn't recognize it and the chatbox is not a real browse window, its byond fake html window.